### ISSUE TITLE
feature-request: Notifications to mastodon

### ISSUE THREAD

**Issue by laanwj** [gkrizek/ghi#8]
I currently run two separate github event notification systems in parallel:

- one based on an old version of [bitcoinacks](https://github.com/PierreRochard/bitcoin-acks)—this can do notifications to Mastodon and Twitter (I only care about the former right now), for merges (polling based)
- `ghi`—for notifying IRC (webhook notification based)

It would be nice to have this in one system, specified by one configuration file.

For comparison here was the patch to add "toot" support to `bitcoinacks`: https://github.com/PierreRochard/bitcoin-acks/commit/9f828d960fd499a2881779156b8adf260efbfe0e

**Comment by laanwj** [comment]
I think the main difference between both approaches, between bitcoin-acks's polling and `ghi`'s webhook based approach, is that the former can 'catch up' to missed notifications when the script was down for a while. But I'm not actually sure this is an important feature. The most important function of the bot (https://x0f.org/@bitcoinmerges) is notification of current merges, not so much to keep a consistent history.
